src/apps/common/hiredis_libevent2.c

```diff
@@ -233,7 +233,7 @@ redis_context_handle redisLibeventAttach(struct event_base *base, char *ip0, int
   if(ip0 && ip0[0])
 	  STRCPY(ip,ip0);
```

Comment:
isn't it here where strncpy matters and not 2 lines below where the IP is fixed lenght?

Same in the next file

Reply:
I was just going after warnings this time
The warning is generated in STRCPY(dst,"string") and not in STRCPY(dst,src)